### ISSUE TITLE
Permission denied on writing in lambda's docker volume

### ISSUE THREAD

**Issue by Nikita-Sherstnev** [getmoto/moto#6192]
Hello! I am running lambda in docker container mlupin/docker-lambda. When I am trying to create directory inside this volume I got permission denied error. How can I set that lambda can write in my volume?

**Comment by bblommers** [comment]
Hi @Nikita-Sherstnev! If the image that you try to run does not exist in `mlupin`, Moto will fall-back to `lambci`. This typically happens if you use an older Python version.

Which directory are you trying to create? Have you verified that it works in AWS, or does AWS also have that directory as write-protected?

**Comment by Nikita-Sherstnev** [comment]
Thank you for hint, I tried to create directory outside of /tmp.
I have another question, is it possible to mount some directory to lambda container? I need to pass input files there, which are generated only in runtime. I would like to avoid usage of S3 to mitigate complexity of tests.

**Comment by bblommers** [comment]
No, there is no builtin way to reach the container/volume. It is pretty much a black box, with very limited input and output.

Note that, if you're using the decorators, other services (like S3) are not reachable from inside a Lambda container. That is only possible when running `moto` as a server - see http://docs.getmoto.org/en/latest/docs/services/lambda.html for details and configuration options.
